hopkin/routes/restaurants.py

```diff
@@ -0,0 +1,105 @@
+import json
+from bson.json_util import dumps
+from flask import Blueprint, jsonify, request, g
+
+restaurant_api = Blueprint('restaurant_Api', __name__)
+
+
+def get_restaurant_as_object(restaurant):
```

Comment:
Is this method still needed if [this is there](https://github.com/project-hopkins/Westworld/pull/23/files#diff-2c8bbadf338e34ccaec2bf66ed18fa3bR2)?

Reply:
No, I will make the switch and push changes